src/Features/Core/Portable/GenerateMember/GenerateConstructor/AbstractGenerateConstructorService.Editor.cs

```diff
@@ -413,7 +413,6 @@ private IEnumerable<string> GetUnavailableMemberNames()
                         var property = (IPropertySymbol)symbol;
                         return
                             property.Parameters.Length == 0 &&
-                            property.SetMethod != null &&
```

Comment:
we should make sure there's *either* a SetMethod *or* a  backing field.

Can you add a test that if there is an *abstract* getter-only property, that we still don't assign to it?